### PR TITLE
Allow customize images for inventory configmap

### PR DESCRIPTION
**Pull request by masayag**
Signed-off-by: Moti Asayag <masayag@redhat.com>

### CODE REVIEW

File: discovery-infra/update_bm_inventory_cm.py [hunk 1]
```diff
@@ -32,6 +32,10 @@
     ("AGENT_DOCKER_IMAGE", ""),
     ("KUBECONFIG_GENERATE_IMAGE", ""),
     ("BASE_DNS_DOMAINS", ""),
+    ("IMAGE_BUILDER", ""),
+    ("CONNECTIVITY_CHECK_IMAGE", ""),
+    ("HARDWARE_INFO_IMAGE", ""),
+    ("INVENTORY_IMAGE", ""),
```
Comment (masayag) [inline discovery-infra/update_bm_inventory_cm.py:38]: @eranco74 could you describe what is the role of INVENTORY_IMAGE ?

Comment (eranco74) [reply discovery-infra/update_bm_inventory_cm.py:38]: It's the bm-inventory image.

Comment (eranco74) [reply discovery-infra/update_bm_inventory_cm.py:38]: I think it's used for the versions API

**Comment by masayag** [comment]
@tsorya Would you like this PR being cherry-picked to master branch ?

**Comment by filanov** [comment]
@masayag how about updating the README with the new options?

**Pull request merged**